src/components/Container.tsx

```diff
@@ -17,14 +17,7 @@ const ContainerBase = styled.div<ThemedContainerProps>`
 `;
 
 const Container = (props: ThemedContainerProps) => {
-	return (
-		<ContainerBase
-			{...props}
-			maxWidth={props.theme.breakpoints.map(
-				(bp, i) => `calc(${bp} - ${props.theme.space[i]}em)`,
-			)}
-		/>
-	);
+	return <ContainerBase {...props} maxWidth={props.theme.containers} />;
```

Comment:
How about like this:
```suggestion
	return <ContainerBase {...props} px={[0, 1, 2, 3]} maxWidth={props.theme.breakpoints} />;
```

Reply:
@sradevski 	The thing is that the common libraries don't apply the breakpoint as the max-width. Bootstrap does that with `container-fluid` instead.